Map Update Train

Dunrab opened the pull request:
# Description

A lot of changes to train based on what our community wanted the map to have.

The following changes have been made:

- expanded dorms to be a better size for privacy and to match dorms styles on other maps
- fixed messed up lattice that caused extra grids to be loaded
- fixed the parallax letting magboots be usable to walk around when you should be affected by no gravity 
- added tiny fans to the chefs freezer
- moved botany to across from the kitchen to give them more space and machines
- old botany area was retro-fitted to hold chickens and the chefs cow
- moved janitors closet across from the kitchen
- added a way to vent the arti chamber for epi
- epi can now access the prober and server with out a mysta present on shift
- expanded mystagogue's bedroom
- redid main medical area to expand and give medical more room to work
- added more medical lockers to the medical locker room
- moved the paramed room to the old moruge
- moved morgue and cloning to the new medical area
- turned old cloning room into a mini virology room for beds
- expanded cryo for medical so they have two pods to use, it is also now hooked up to distro instead of using a singular air can
- medifab has been moved into medical along with all of their medkits, medifab on shift start will have 30 glass, steel and plastic (can tweak this if we think its to much)
- added airlocks/tiny fans in medical maints to the broken train car to stop the constant freezing air from coming in
- expanded and re-made perma
- expanded LO's office slightly by moving the logi substation
- all heads have ID pcs in their office
- swapped out the recycler for a materail reclaimer in disposals
- added full justice department
- turned lawyer room that was in the bar into a little lounge room
- turned the old jani room into a bar expansion
- moved reporters office to give them a bigger space
- moved corpsman room further up
- expanded hos's bedroom so its not a tiny cube
- moved the secfab so that any officer can access it with out having to rely on hos/warden
- made it so sec officers can acccess four of the hardsuits if they need to, there are three more in the armory for warden/hos to give out
- station did not have a disabler safe so added one to the sec locker room
- expanded LO's room and gave them a bedroom
- expanded the CE's room and gave them a proper bedroom
- added a few train cars that can be refurbished or repurposed into whatever someone would like to make, probably will want to add a few more
- atmos outside of the station is now -60 Celsius, it was previously -120 Celsius which was a bit to high

---

# TODO

- [ ] should blue shield and assistant be mapped?
- [ ] remake the chapel, perhaps give the chaplin an office probably will need to expand the that train car in order to do fit that in.
- [ ] better evac for train 
- [ ] where to put evac yml file for train
- [ ] see if there is anything else that should get added/tweaked
- [ ] test cryo to make sure the changes work properly
- [ ] check station beacons, moved a lot of room around so its possible I forgot some or some are in bad places and will make it hard to read
---

<details><summary><h1>Map Renders</h1></summary>
<p>


![Train-0](https://github.com/user-attachments/assets/c4b1aa1a-aa1a-4fe3-a63e-0c696640492a)


</p>
</details>

---

# Changelog

:cl:
- tweak: Tweaked Train based on community feedback and requests. This includes adding the full Justice department.

Dunrab committed:
train updates

fixed messed up lattice that caused extra grids to be loaded
expanded dorms to be a better size for privacy and to match dorms styles on other maps
added tiny fans to the chefs freezer
moved botany to across from the kitchen to give them more space and machines
old botany area was retro fitted to hold chickens and the chefs cow
moved janitors closet across from the kitchen
added a way to vent the arti chamber for epi
epi can now access the prober and server with out a mysta present on shift
expanded mystagogue's bedroom
redid main medical area to expand and give medical more room to work
added more medical lockers to the medical locker room
moved the paramed room to the old morgue
moved morgue and cloning to the new medical area
turned old cloning room into a mini virology room for beds
expanded cryo for medical so they have two pods to use, it is also now hooked up to distro instead of using a singular air can
added airlocks/tiny fans in medical maints to the broken train car to stop the constant freezing air from coming in
expanded and re-made perma
expanded LO's office slightly by moving the logi substation
all heads have ID pcs in their office
swapped out the recycler for a material reclaimer in disposals
added full justice department
turned lawyer room that was in the bar into a little lounge room
turned the old jani room into a bar expansion
moved reporters office to give them a bigger space
moved corpsman room further up
expanded hos's bedroom so its not a tiny cube
moved the secfab so that any officer can access it with out having to rely on hos/warden
made it so sec officers can access four of the hardsuits if they need to, there are three more in the armory for warden/hos to give out
station did not have a disabler safe so added one to the sec locker room
expanded LO's room and gave them a bedroom
expanded the CE's room and gave them a proper bedroom

Dunrab committed:
add justice department job slots

add justice department job slots

Raikyr0 commented:
Hmm, the secret hidey spot at the very end of the train got removed.

Dunrab committed:
add an additional paramed slot

add an additional paramed slot, i expand the paramed room to fit two parameds so need to give another slot

Dunrab commented:
> Hmm, the secret hidey spot at the very end of the train got removed.

that spot was mostly for syndis to help caught syndis escape perma on wizden sine it was just two walls that had to be welded to help them break out. Since antags aren't as important here on floof i used the space to make perma more enjoyable for the people that like to take that role

Dunrab commented:
was it ever decided if we wanted to have blue shield and admin assistant mapped or did we want to wait till we got all of their gear and stuff? If we do then ill add them in since I'm already updating train

southbridge-fur commented:
This is looking great! I'll do a full review in a bit. 

Any chance I could convince you to also separate the TEG burn chamber so it's surrounded by space on all sides?

Dunrab commented:
> This is looking great! I'll do a full review in a bit.
> 
> Any chance I could convince you to also separate the TEG burn chamber so it's surrounded by space o ball sides?

sure thing, ill wait for anything else you bring up and add it to that list so i can get it all done in one go

Dunrab committed:
additional train updates

expanded the bar and added more side cars

Dunrab commented:
I wanted to work on a map specific evac for train as well so it can better dock to station, my question is where do we in /maps/floof/ do we want to have those stored? I'm under the assumption we don't want to just stick in the Shuttle folder

Dunrab committed:
expanded evac dock

expanded the evac dock so the evac shuttle can more easily dock and not have issues docking
